src/schema/v2/ArtworkImport/artworkImport.ts

```diff
@@ -109,16 +110,21 @@ const ArtworkImportRowConnectionType = connectionWithCursorInfo({
   nodeType: ArtworkImportRowType,
 }).connectionType
 
-export const ArtworkImportType = new GraphQLObjectType({
+export const ArtworkImportType = new GraphQLObjectType<any, ResolverContext>({
   name: "ArtworkImport",
   interfaces: [NodeInterface],
-  fields: {
+  fields: () => ({
     ...InternalIDFields,
     columns: {
       type: new GraphQLNonNull(GraphQLList(new GraphQLNonNull(GraphQLString))),
       description:
         "Columns to display for an import, will exist in a row's `transformedData`",
     },
+    createdAt: date(),
+    createdBy: {
```

Comment:
This doesn't match the `created_by` in the Gravity PR - https://github.com/artsy/gravity/pull/18639 - where this is a `User`.

Reply:
Yes, I had already raised this before making changes in Gravity. This has been updated in https://github.com/artsy/metaphysics/pull/6475/commits/30dce98dd591a1e6335fd02b8a95ee4e12d1442b